templates/common/_base/files/clean-ovs-network.yaml

```diff
@@ -0,0 +1,60 @@
+mode: 0755
+path: "/usr/local/bin/clean-ovs.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+
+    # Revert changes made by /usr/local/bin/configure-ovs.sh.
+    # Remove OVS bridge "br-ex". Use the default NIC for cluster network.
+
+    iface=""
+    counter=0
+    # find default gateway interface
+    while [ $counter -lt 12 ]; do
```

Comment:
This should not be necessary; the setup side needs to deal with the possibility that the first time it runs, NM might be partway through bringing the network up for the first time, and so we don't know which interface will end up being the default. But in the cleanup case, either `br-ex` and `ovs-port-phys0` and the rest already exist as NetworkManager objects, and you know what needs to be done with them; or else they don't exist, and there's no cleanup to be done.

Reply:
fixed.